### PR TITLE
Added Peter F Hamiltons "Salvation Sequence".

### PR DESCRIPTION
**Pull request by nahkampf**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -825,6 +825,13 @@ Alastair Reynolds' take on the Fermi Paradox is a vast and brutal epic, consisti
 
 > Nine hundred thousand years ago, something annihilated the Amarantin civilization just as it was on the verge of discovering space flight. Now one scientist, Dan Sylveste, will stop at nothing to solve the Amarantin riddle before ancient history repeats itself. With no other resources at his disposal, Sylveste forges a dangerous alliance with the cyborg crew of the starship Nostalgia for Infinity. But as he closes in on the secret, a killer closes in on him. Because the Amarantin were destroyed for a reason — and if that reason is uncovered, the universe—and reality itself — could be irrecoverably altered...
 
+#### [Salvation](https://www.goodreads.com/book/show/34068552-salvation) (2018-2020) *by [https://en.wikipedia.org/wiki/Peter_F._Hamilton)* [4.19 avg]
```
Comment (RichardLitt) [inline readme.md:828]: Peter Hamilton link looks malformed, both in this book and the one above. Can you fix them?

Comment (nahkampf) [reply readme.md:828]: Both of these are as provided by their services, and they resolve. It doesn't for you?

Comment (RichardLitt) [reply readme.md:828]: The brackets. `[)` isn't right.

Comment (nahkampf) [reply readme.md]: Ah, doh, I'm blind. And I copied the pertinent info from the other Hamilton entry, of course that one was also wrong :) Fixed

Comment (RichardLitt) [reply readme.md]: It's alright, that's why we review PRs. :)

**Comment by RichardLitt** [comment]
Thanks!

**Pull request merged**